types/has-symbols/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["ES2015"],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": ["../"],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
+    },
+    "files": [
+        "has-symbols-tests.ts",
```

Comment:
Needs to include shams.d.ts

Reply:
That’s not been allowed since <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40676> (<https://github.com/microsoft/types-publisher/issues/708>).